docs/kss/starter-pages/details-1.html

```diff
@@ -0,0 +1,367 @@
+<!DOCTYPE html>
+
+<head>
+  
+  <title>Techne: Layout Template - Table</title>
+
+  <meta name="viewport" content="width=device-width">
+
+  <link rel="stylesheet" href="../public/dist/css/techne.min.css">
+  <link rel="stylesheet" href="css/demo.css">
+
+</head>
+
+<body>
+
+    <!-- page header  -->
+    <div class="navbar navbar-default y-navbar-centered y-navbar-lg">
+    	<div class="container-fluid">
+    		<div class="navbar-left">
+    			<div class="y-toolbar">
+    				<div class="y-toolbar-cell">
+    					<button class="btn-subordinate btn navbar-btn">
+    						<span class="hyicon hyicon-back"></span> <span class="hidden-xs">Back</span>
+    					</button>
+    				</div>
+    			</div>
+    		</div>
+    		<div class="y-navbar-center">
+    			<div class="y-navbar-center-content">
+    				<h1>Page Title</h1>
+    			</div>
+    		</div>
+    		<div class="navbar-right">
+            <!-- 							
+            	<div class="y-toolbar">
+    				<div class="y-toolbar-cell">
+    					<button class="btn btn-subordinate navbar-btn">Cancel
+    					</button>
+    				</div>
+    				<div class="y-toolbar-cell">
+    					<button class="btn btn-warning navbar-btn">Save
+    					</button>
+    				</div>
+    			</div> 
+    		-->
+    		</div>
+    	</div>
+    </div>
+            
+    <!-- page level help -->
+    <span class="page-level-help-message">Page level help message goes here</span>
+    
+    <!-- top panel -->
+    	
+    <div class="panel panel-default">
+    	<div class="panel-body">
+    		<div class="container">
```

Comment:
it looks like we're using two different types of tabs here.  please set to the same